src/commands/babel.js

```diff
@@ -24,6 +27,8 @@ export default class Babel {
             const { code } = transform(es6, {
                 presets: [preset],
                 babelrc: false
+
+                
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.whitespace`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp49z0c0qn/src/commands/babel.js
+++ b/tmp/tmp49z0c0qn/src/commands/babel.js
@@ -28,7 +28,7 @@
                 presets: [preset],
                 babelrc: false
 
-                
+
             })
 
             return Rin.code('js', code)
```